docs/reports/migrate.md

```diff
@@ -0,0 +1,14 @@
+# Slurp: Migratable terms
+| Ontology                          | Tot   |
+|:----------------------------------|:------|
+| [DOID](./migrate_doid.md)         | 678   |
+| [NCIT](./migrate_ncit.md)         | 5,516 |
+| [ORDO](./migrate_ordo.md)         | 27    |
+| [OMIM](./migrate_omim.md)         | 225   |
+| [ICD10WHO](./migrate_icd10who.md) | 117   |
+| [ICD10CM](./migrate_icd10cm.md)   | 1,851 |
+
+`Ontology`: Name of ontology    
+`Tot`: Total terms migratable
```

Comment:
What exactly is the content of migratable? Unmapped mappable? Matched unmapped mappable?

Reply:
I had an existing comment in `migrate.py` which explains this. I decided to rephrase it, and I have added it to the Jinja for `docs/reports/migrate.md`, so this text will appear the next time the goal is run. I also added some subheadings:

```md
# Slurp: Migratable terms
{{ stats_markdown_table }}

### Codebook
`Ontology`: Name of ontology    
`Tot`: Total terms migratable

### Definitions
**Migratable term**: A term owned by the given ontology but unmapped in Mondo, which has either no parents, or if it has 
parents, all of its parents have already been mapped in Mondo.

### Workflows
To run the workflow that creates this data, refer to the [workflows documentation](../developer/workflows.md).
```

Reply:
Thanks!